pulsar-functions/worker/src/main/java/org/apache/pulsar/functions/worker/FunctionRuntimeManager.java

```diff
@@ -384,7 +385,7 @@ public synchronized void restartFunctionInstance(String tenant, String namespace
         }
     }
 
-    public synchronized void restartFunctionInstances(String tenant, String namespace, String functionName)
+    public synchronized void restartFunctionInstances(String tenant, String namespace, String functionName, ComponentType componentType)
```

Comment:
how about we get `ComponentType` from `Assignment.instance.functionMetaData.functionDetails.ComponentType` instead of passing it as an argument?